core/src/action.rs

```diff
@@ -65,15 +68,18 @@ impl Hash for ActionWrapper {
 pub enum Action {
     /// entry to Commit
     /// MUST already have passed all callback checks
-    Commit(Entry),
-    /// hash to Get
-    Get(String),
-
+    CommitEntry(Entry),
+    /// hash of Entry to get
+    GetEntry(HashString),
+    /// entry to Commit
+    /// MUST already have passed all callback checks
+    LinkAppEntries(Link),
+    /// hash to Entry to get links from
+    GetLinks(GetLinksArgs),
```

Comment:
ditto for `Links`

Reply:
Was sticking to hdk: `hdk::get_links()` but maybe we should change to `hdk::links` in the HDK also?